spring/boot-actuator-autoconfigure/build.gradle

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
We usually do not put this Copyright on gradle files?

Reply:
IntelliJ automatically adds it when using the imported settings from what I can tell - do we want to have contributors manually remove it? I can remove if it's better, but not sure if there's a reason to exclude just one type of file.

Reply:
Never knew IntelliJ IDEA adds license header to a build file. Our convention so far was not to add license header to a build file, so it'd be nice to remove it. 🙏

Reply:
🙏

Reply:
Ok no worries - removed the license headers

Reply:
To add just a little bit, I think it is somewhat weird to claim copyright on build scripts because it often is meant to be open for copying and pasting. 😅